PostHog/PostHogRemoteConfig.swift

```diff
@@ -442,6 +456,70 @@ class PostHogRemoteConfig {
         return flags
     }
 
+    func setPersonPropertiesForFlags(_ properties: [String: Any]) {
+        personPropertiesForFlagsLock.withLock {
+            // Merge properties additively, similar to JS SDK behavior
+            personPropertiesForFlags.merge(properties, uniquingKeysWith: { _, new in new })
+            // Persist to disk
+            storage.setDictionary(forKey: .personPropertiesForFlags, contents: personPropertiesForFlags)
+        }
+    }
+
+    func resetPersonPropertiesForFlags() {
+        personPropertiesForFlagsLock.withLock {
+            personPropertiesForFlags.removeAll()
+            // Clear from disk
+            storage.setDictionary(forKey: .personPropertiesForFlags, contents: personPropertiesForFlags)
```

Comment:
i think this should be centralized here https://github.com/PostHog/posthog-ios/blob/03803c80ae67695667a8106c98af92caac94e81f/PostHog/PostHogStorage.swift#L352
same for groups